lib/associations/mixin.js

```diff
@@ -209,18 +209,27 @@ const Mixin = {
     return association;
   },
 
-  getAssociation(target, alias) {
```

Comment:
I assumed that `getAssociation` is a private API. I effectively replaced it and all of it's other two invocations with `getAssociationForAlias`